File: 09_CoreAnimation_立方体+旋转_OC/08_GLKit_OC/ViewController.m
//
//  ViewController.m
//  08_GLKit_OC
//
//  Created by 陈嘉琳 on 2020/7/24.
//  Copyright © 2020 CJL. All rights reserved.
//

#import "ViewController.h"
#import "CAViewController.h"


@interface ViewController ()


@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
   
}
- (IBAction)btnClick:(id)sender {
    CAViewController *vc = [[CAViewController alloc] init];
    [self presentViewController:vc animated:true completion:nil];
    
}



@end

File: 09_CoreAnimation_立方体+旋转_Swift/GLKit_demo/ViewController.swift
 //
 //  ViewController.swift
 //  GLKit_demo
 //
 //  Created by  on 2020/7/25.
 //

 import UIKit


 class ViewController: UIViewController{
     

     override func viewDidLoad() {
         super.viewDidLoad()
         
         self.view.backgroundColor = UIColor.white
         
        
     }
    
    @IBAction func btnClick(_ sender: Any) {
        
        let vc = CAViewController.init()
        self.present(vc, animated: true, completion: nil)
        
    }
    
     
 }


